### ISSUE TITLE
Some Python warnings in QGIS 3.4.0

### ISSUE THREAD

**Issue by agiudiceandrea** [nextgis/quickmapservices#179]
```
2018-11-01T13:33:11     WARNING    warning:C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py:672: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
              mod = _builtin_import(name, globals, locals, fromlist, level)
             
             traceback: File "", line 1, in 
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 335, in startPlugin
              plugins[packageName] = package.classFactory(iface)
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\__init__.py", line 34, in classFactory
              from .quick_map_services import QuickMapServices
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\quick_map_services.py", line 40, in 
              from .qgis_map_helpers import add_layer_to_map
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qgis_map_helpers.py", line 17, in 
              from .py_tiled_layer.tilelayer import TileLayer
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\py_tiled_layer\tilelayer.py", line 44, in 
              from osgeo import gdal
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\osgeo\__init__.py", line 21, in 
              _gdal = swig_import_helper()
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\osgeo\__init__.py", line 8, in swig_import_helper
              import imp
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\imp.py", line 33, in 
              DeprecationWarning, stacklevel=2)
             
2018-11-01T13:33:13     WARNING    warning:C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\status_codes.py:18: DeprecationWarning: invalid escape sequence \o
              """
             
             traceback: File "", line 1, in 
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 335, in startPlugin
              plugins[packageName] = package.classFactory(iface)
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\__init__.py", line 34, in classFactory
              from .quick_map_services import QuickMapServices
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\quick_map_services.py", line 41, in 
              from .qms_service_toolbox import QmsServiceToolbox
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_service_toolbox.py", line 47, in 
              from .qms_external_api_python.client import Client
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_external_api_python\client.py", line 2, in 
              from .api.api_v1 import ApiClientV1 as Client
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_external_api_python\api\api_v1.py", line 1, in 
              from .api_abstract import ApiClient
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_external_api_python\api\api_abstract.py", line 4, in 
              from requests import get
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\__init__.py", line 114, in 
              from .models import Request, Response, PreparedRequest
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\models.py", line 43, in 
              from .status_codes import codes
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 724, in exec_module
              File "", line 860, in get_code
              File "", line 791, in source_to_code
              File "", line 219, in _call_with_frames_removed
             
2018-11-01T13:33:14     WARNING    warning:C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\objcreator.py:152: DeprecationWarning: 'U' mode is deprecated
              plugin = open(filename, 'rU')
             
             traceback: File "", line 1, in 
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 335, in startPlugin
              plugins[packageName] = package.classFactory(iface)
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\__init__.py", line 34, in classFactory
              from .quick_map_services import QuickMapServices
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\quick_map_services.py", line 41, in 
              from .qms_service_toolbox import QmsServiceToolbox
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 672, in _import
              mod = _builtin_import(name, globals, locals, fromlist, level)
              File "", line 983, in _find_and_load
              File "", line 967, in _find_and_load_unlocked
              File "", line 677, in _load_unlocked
              File "", line 728, in exec_module
              File "", line 219, in _call_with_frames_removed
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_service_toolbox.py", line 124, in 
              path.dirname(__file__), 'qms_service_toolbox.ui'))
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\__init__.py", line 198, in loadUiType
              winfo = compiler.UICompiler().compileUi(uifile, code_string, from_imports, resource_suffix, import_from)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\Compiler\compiler.py", line 53, in __init__
              qtproxies.QtWidgets, CompilerCreatorPolicy())
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\uiparser.py", line 177, in __init__
              self.factory = QObjectCreator(creatorPolicy)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\objcreator.py", line 85, in __init__
              if self.load_plugin(filename, plugin_globals, plugin_locals):
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\PyQt5\uic\objcreator.py", line 152, in load_plugin
              plugin = open(filename, 'rU')
             
2018-11-01T13:33:15     WARNING    warning:C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\urllib3\connectionpool.py:857: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
              InsecureRequestWarning)
             
             traceback: File "", line 1, in 
              File "C:/PROGRA~1/QGIS3~1.4/apps/qgis/./python\qgis\utils.py", line 344, in startPlugin
              plugins[packageName].initGui()
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\quick_map_services.py", line 121, in initGui
              self.init_server_panel()
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\quick_map_services.py", line 311, in init_server_panel
              self.server_toolbox = QmsServiceToolbox(self.iface)
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_service_toolbox.py", line 155, in __init__
              self.show_news()
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_service_toolbox.py", line 160, in show_news
              qms_news = client.get_news()
              File "C:/Users/---------/AppData/Roaming/QGIS/QGIS3\profiles\test/python/plugins\quick_map_services\qms_external_api_python\api\api_abstract.py", line 64, in get_news
              response = get(url, verify=False)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\api.py", line 72, in get
              return request('get', url, params=params, **kwargs)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\api.py", line 58, in request
              return session.request(method=method, url=url, **kwargs)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\sessions.py", line 512, in request
              resp = self.send(prep, **send_kwargs)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\sessions.py", line 622, in send
              r = adapter.send(request, **kwargs)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\requests\adapters.py", line 445, in send
              timeout=timeout
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\urllib3\connectionpool.py", line 600, in urlopen
              chunked=chunked)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\urllib3\connectionpool.py", line 343, in _make_request
              self._validate_conn(conn)
              File "C:\PROGRA~1\QGIS3~1.4\apps\Python37\lib\site-packages\urllib3\connectionpool.py", line 857, in _validate_conn
              InsecureRequestWarning)
             
```

**Comment by 514ckw4r3** [comment]
@agiudiceandrea, please verify this issue in the last release

**Comment by agiudiceandrea** [comment]
Yes, fixed now (version 0.19.11). Thanks.
